misc/config_tools/service_vm_config/serial_config.py

```diff
@@ -37,20 +37,6 @@ def main(args):
                 user_vm_id = legacy_vuart.find('target_vm_id').text
                 legacy_vuartid = int(legacy_vuart.attrib["id"])
                 vuart_target_vmid[legacy_vuartid] = user_vm_id
-
-    vm_list = allocation_etree.xpath("//vm[load_order = 'SERVICE_VM']")
-    for vm in vm_list:
-        vuart_list = find_non_standard_uart(vm)
```

Comment:
Does this patch remove the generation of the `serial.conf` file that we use to expose non-standard serial port in the Service VM?

Reply:
We have not removed the support for non-standard serial port. 
Users could set non-standard UART by the new UI, IRQ is generated automatically, and io port use a default value.

Reply:
But there is no need for the `serial.conf` file ever? The Service VM will always see whatever serial port the user sets via the UI, is that correct?

Reply:
> 

Can you please answer this?

Reply:
I have check again, now our vuart driver can't create non-stand vuart.
So the SOS still need the serial.conf.

But the logic of create serial.conf can't adapter the new vuart connection, so these logic should update.
There will be another patch to fix this.

It's a great honor to invite you to review these patch.